book.rb

```diff
@@ -0,0 +1,16 @@
+require './rentals'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def add_rentals=(person, date)
+    Rental.new(self, person, date)
+  end
```

Comment:
- Please note that this method should implement the `has-many` relationship between the `Book` class and `Rental` class, it does not serve as a setter method. Please fix this 🙏